vue/mixins/blob.js

```diff
@@ -0,0 +1,22 @@
+export const blobMixin = {
+    methods: {
+        /**
+         * Downloads a blob file with a given filename
+         *
+         * @param {Object} blob File to be downloaded
+         * @param {String} filename Name to give the file
+         */
+        downloadBlob(blob, filename) {

```

Comment:
This can actually be used with `File`, `Blob` or `MediaSource` so just name it `download`

Reply:
Because of this, rename to `downloadMixin` as well.

See: https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL